packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -214,7 +214,8 @@ export const EventMeta = ({
                     menuPosition="absolute"
                     timezoneSelectCustomClassname={classNames?.eventMetaTimezoneSelect}
                     classNames={{
-                      control: () => "!min-h-0 p-0 w-full border-0 bg-transparent focus-within:ring-0",
+                      control: () =>
+                        "!min-h-0 p-0 w-full border-0 bg-transparent focus-within:ring-0 shadow-none!",
```

Comment:
```suggestion
                        "!min-h-0 p-0 w-full border-0 bg-transparent focus-within:ring-0 !shadow-none",
```

! should go to the beginning, right?

Reply:
Yes :P